src/business/BookService.java

```diff
@@ -0,0 +1,29 @@
+package business;
+
+import dataaccess.BookRepo;
+import dataaccess.BookRepoAccess;
+
+public class BookService {
+	BookRepoAccess bookRepo = new BookRepo();
+	boolean createBookCopy(Book book) throws LibrarySystemException {
```

Comment:
method should be `public` and accept parameters needed to construct book instance inside the method